pjsip-apps/src/samples/pcaputil.c

```diff
@@ -54,6 +54,7 @@ static const char *USAGE =
 "                           AES_CM_128_HMAC_SHA1_80 \n"
 "                           AES_CM_128_HMAC_SHA1_32\n"
 "  --srtp-key=KEY, -k     Set the base64 key to decrypt SRTP packets.\n"
+"  --codec-fmtp=FMTP      Set the fmtp input for parsing codec options.\n"
```

Comment:
Just provide an example here since it may not be obvious that it can contain multiple parameters separated by semi-colon.
For example: `'mode-set=0;octet-align=1'

Reply:
Done